docs/compose.md

```diff
@@ -131,5 +83,5 @@ backend as part of the Dockerfile. Then use Docker to create the
 image.
 
 ```
-$ docker build -t tritonserver_custom .
+$ docker build -t tritonserver_custom -f Dockerfile.compose .
```

Comment:
Remove $ as it is removed from other places

Reply:
Other stuff is from the Dockerfiles. Would it be more clear if we have the `$` to indicate its a bash command?

Reply:
I don't think it is necessary to have $. It should be clear. In fact you removed it [elsewhere](https://github.com/triton-inference-server/server/pull/3039/files/e6b3371694294fb60ff21385bafd3bbd391348c1#diff-13411a91b1a348eab8a65b80e6d6709ca1530ab8217d53df8b90fbb07552eb8cR54) in this PR

Reply:
I think then we should add back the `$` in elsewhere, the convention is using `$` to indicate a bash command (see other places in docs)